packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -479,6 +489,13 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
       upperBound: CupertinoContextMenu.animationOpensAt,
     );
     _openController.addStatusListener(_onDecoyAnimationStatusChange);
+    _openController.addListener(() {
+      if (_openController.status != AnimationStatus.reverse &&
+          (_openController.value * 1000).truncate() == 500 &&
```

Comment:
I tested this code on a physical device to be sure, but this is rarely going to trigger as the listener callback for the animation controller isn't going to fire for every value, so it's rarely going to hit 0.500 exactly. I'd guess the best way to do this would be to move the callback to it's own variable, then add it on tap, and remove it both on release, as well as at the end of the callback itself, after the feedback is triggered.

Reply:
That actually makes a lot of sense. Fix is pushed.